setup.py

```diff
@@ -23,10 +23,10 @@ def requirements():
     license='BSD',
     packages=find_packages(),
     install_requires=[
-        'fairseq',
+        'fairseq>=0.5.0',
     ],
     dependency_links=[
-        'git+https://github.com/pytorch/fairseq.git#egg=fairseq',
+        'git+https://github.com/pytorch/fairseq.git#egg=fairseq-0.5.0',
```

Comment:
@myleott, so, line 26 seems good but is this change (line 29) necessary?

Reply:
Yes, unfortunately dependency links don't get associated with version numbers unless they are explicitly added to the URI. Without this change, `setup.py install` will complain about not being able to find 0.5.0. Without line 26 it won't properly update if you have 0.4.0 :/

Btw, this should still pull the latest master commit of fairseq, the version numbers are only loosely enforced by setuptools.